packages/engine/src/actions/link.js

```diff
@@ -16,14 +16,14 @@
 
 import { type } from '@lowdefy/helpers';
 
-async function Link({ context, params }) {
+const link = ({ context, params }) => {
   const linkParams = type.isString(params) ? { pageId: params } : params;
```

Comment:
@Gervwyk I think this logic should move to the action function, and the `link` method should just be link method on context?

Reply:
Yes, I agree. `link` should just expose the internal `link` method here. So this overload logic and error can go to the action. The internal `link` method will throw an "invalid link" error, so the action here just adds some more context by catching the error. Consider this also regarding the comment of where the error should be thrown.